ingestion-beam/pom.xml

```diff
@@ -49,32 +49,14 @@
     <dependencyManagement>
         <dependencies>
             <!-- Keep these dependency versions in sync with what is pulled in by beam;
-                 check https://mvnrepository.com/artifact/org.apache.beam/beam-sdks-java-io-google-cloud-platform/2.22.0
-
-                 Does not use libraries-bom to determine versions as of beam 2.19 because the lowest
-                 version available uses google-cloud-bom 0.84.0-alpha, but 0.79.0-alpha is needed to
-                 match what is pulled in by beam
-
-                 Does not use google-cloud-bom 0.79.0-alpha to determine versions as of beam 2.19
-                 because that causes integration tests to fail on dataflow with
-                 java.lang.NoSuchMethodError and java.lang.NoClassDefFoundError
-
-                 These versions are not shared via a property, because they are independently
-                 versioned and more recent versions do not match -->
-            <dependency>
-                <groupId>com.google.cloud</groupId>
-                <artifactId>google-cloud-bigquery</artifactId>
-                <version>1.61.0</version>
-            </dependency>
+                 check https://mvnrepository.com/artifact/org.apache.beam/beam-sdks-java-io-google-cloud-platform/2.28.0
+                 -->
             <dependency>
                 <groupId>com.google.cloud</groupId>
-                <artifactId>google-cloud-pubsub</artifactId>
-                <version>1.61.0</version>
-            </dependency>
-            <dependency>
-                <groupId>com.google.cloud</groupId>
-                <artifactId>google-cloud-storage</artifactId>
-                <version>1.61.0</version>
+                <artifactId>libraries-bom</artifactId>
+                <version>13.2.0</version>
+                <type>pom</type>
+                <scope>import</scope>
```

Comment:
beam 2.28 uses libraries-bom 13.2.0, and this requires a slightly newer version of the google-cloud-pubsub package